roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -1181,13 +1181,20 @@ impl Device for TweezerDevice {
     }
 
     fn two_qubit_edges(&self) -> Vec<(usize, usize)> {
+        let tw_two_qubit_gate_times = &self
+            .get_current_layout_info()
+            .unwrap()
```

Comment:
Is this unwrap safe?

Reply:
Technically no, it isn't, but qoqo Device trait does not return Results so there's no other good way to handle it (if you spot it let me know).

Reply:
Could you change this to an `expects` with a verbose error message?